src/serial.rs

```diff
@@ -734,6 +734,32 @@ usart! {
     USART5: (usart5, apb1enr, usart5en, apb1_clk, Serial5Ext),
 }
 
+impl Serial<LPUART1> {
+    pub fn use_lse(&self, rcc: &mut Rcc, _: &LSE) {
+        //Disable transmitter
+        self.usart.cr1.modify(|_, w| w.te().disabled());
+        while self.usart.isr.read().tc().bit_is_clear() {}
+
+        //Disable LPUART1
+        self.usart.cr1.modify(|_, w| w.ue().disabled());
+
+        //Reconfigure LPUART to use LSE
+        rcc.rb.ccipr.modify(|_, w| w.lpuart1sel().lse());
+
+        //Recalculate baudrate
+        //TODO requested baudrate value from the config should be stored somehow and used here
+        //but at the moment hardcoded 9600 will be used
```

Comment:
I think this is fine for now, but it should be mentioned in the documentation of the method.

Reply:
Looks like method needs documentation too  👍